lib/main.dart

```diff
@@ -13,103 +14,319 @@ class MyApp extends StatelessWidget {
+        unselectedItemColor: Colors.grey,
+        items: <BottomNavigationBarItem>[
+          const BottomNavigationBarItem(
+            icon: Icon(Icons.home),
+            label: '',
+          ),
+          BottomNavigationBarItem(
+            icon: Image.asset('assets/bolt.png', width: 30, height: 30),
+            label: '',
+          ),
+          const BottomNavigationBarItem(
+            icon: Icon(Icons.search),
+            label: '',
+          ),
+          BottomNavigationBarItem(
+            icon: Image.asset('assets/menu.png', width: 30, height: 30),
+            label: '',
+          ),
+        ],
+        iconSize: 35,
+        showSelectedLabels: false,
       ),
-      floatingActionButton: FloatingActionButton(
-        onPressed: _incrementCounter,
-        tooltip: 'Increment',
-        child: const Icon(Icons.add),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
     );
   }
 }
+
+Widget _player(BuildContext context) {
+  double width1 = 90;
+  double width2 = 80;
+  if (MediaQuery.of(context).orientation == Orientation.landscape) {
+    width1 = 250;
+    width2 = 260;
+  }
+  return Positioned(top: 545, width: 500, child: Container(
+      color: const Color.fromRGBO(54, 54, 62, 1),
+      width: double.maxFinite,
+      child: Row(
+        children: [
+          IconButton(
+              onPressed: () {},
+              icon: const Icon(Icons.play_arrow, color: Colors.white)),
+          Container(width: width1),
+          Column(
+            children: const [
+              Text(
+                'rain w/ t1de (s&r)',
+                style: TextStyle(
+                    fontSize: 15,
+                    color: Colors.white,
+                    fontWeight: FontWeight.w500),
+              ),
+              Text(
+                'sindye',
+                style: TextStyle(
+                    fontSize: 13,
+                    color: Colors.grey,
+                    fontWeight: FontWeight.w100),
+              ),
+            ],
+          ),
+          Container(width: width2),
+          IconButton(
+              onPressed: () {},
+              icon: const Icon(Icons.favorite, color: Colors.white)),
+        ],
+      )));
+}
+
+Widget _relatedSongsList(BuildContext context) {
+  String title = 'More of what you like';
+  String description = 'Suggestions based on what you\'ve liked or played';
+  var songsImages = [
+    'assets/1_1.jpg',
+    'assets/1_2.jpg',
+    'assets/1_3.jpg',
+    'assets/1_4.jpg'
+  ];
+  var songsTitles = [
+    'Your Pink Hair (...',
+    'Digital Love (Fea...',
+    'reach you + tide...',
+    'Sony Ericsson'
+  ];
+  String songTheme = 'Related tracks';
+  return _songsList(
+      context, title, description, songsImages, songsTitles, songTheme);
+}
+
+Widget _chartSongsList(BuildContext context) {
+  String title = 'Charts: Top 50';
+  String description = 'The most played tracks on SoundCloud this week';
+  var songsImages = [
```

Comment:
Maybe better to redo your data structure as a object, and work with **map** in future

Reply:
Yeah, thanks, though amount of items in scrollers was thought as constant number (4) still map usage seems cleaner and reusable, thank you!